share/spack/gitlab/cloud_pipelines/stacks/ml-cpu/spack.yaml

```diff
@@ -29,8 +29,8 @@ spack:
 
     # JAX
     # https://github.com/google/jax/issues/12614
```

Comment:
If this works, you can remove this issue link, and we can close that issue.

Reply:
removed, also dealt with the issue in tensorflow